### ISSUE TITLE
【深度学习基础】第二十三课：Batch Normalization - welcome to x-jeff blog

### ISSUE THREAD

**Issue by x-jeff** [x-jeff/GitTalk#111]
http://shichaoxin.com/2020/04/28/%E6%B7%B1%E5%BA%A6%E5%AD%A6%E4%B9%A0%E5%9F%BA%E7%A1%80-%E7%AC%AC%E4%BA%8C%E5%8D%81%E4%B8%89%E8%AF%BE-Batch-Normalization/ 

 xinshichao's blog
